src/editing_tools/AdjustTool.vala

```diff
@@ -216,10 +214,7 @@ public class EditingTools.AdjustTool : EditingTool {
         }
 
         public override bool compress (Command command) {
-            AdjustResetCommand reset_command = command as AdjustResetCommand;
-            if (reset_command == null) {
-                return false;
-            }
+            var reset_command = (AdjustResetCommand) command;
```

Comment:
Can it be ensured that `command` is not null and is of type `AdjustResetCommand`?
A safer ways is:
```suggestion
            if (command == null || !(command is AdjustResetCommand)) {
                return false;
            }
            
            var reset_command = (AdjustResetCommand) command;
```

Or maybe instead add a `requires` clause to the function signature if there is no legitimate reason for `command` not to be of the expected type.

Reply:
I think this is enough. Or should I use a requires clause instead?